### PR TITLE
Update Configuration.md

### PR DESCRIPTION
**Pull request by jaklke**
Split up the Network Shares section to distinguish between different panes
image

### CODE REVIEW

**Comment by MariekeGO** [comment]
@jaklke, I've reviewed your PR and made several changes. Could you confirm if this is still OK to merge?

File: user-guide/Standard_Apps/IDP/Advanced_configuration/Configuration.md [hunk 1]
```diff
@@ -8,10 +8,46 @@ The *Configuration* subtab of the *Admin* tab consists of the pages detailed bel
...
+- **Password**: The password corresponding with the username.
+
+> [!NOTE]
+> Prior to IDP 1.1.20, the file transfer credentials are mentioned under *Working Directories*.
+
+### Working Directories
 
 In the working directories list, you can specify a working directory on the network shares for each hosting DMA. Working directories temporarily contain files during a configuration update. A working directory must be available for each DMA in the DMS.
 
+For each working directory in the DMS, the following settings need to be configured:
```
Comment (jaklke) [inline user-guide/Standard_Apps/IDP/Advanced_configuration/Configuration.md:44]: In the below list, the Agent Name and Directory Status are actually not something that should be configured. My mistake.

**Review by jaklke — COMMENTED** [review]
Please check comment in section about working directories.

File: user-guide/Standard_Apps/IDP/Advanced_configuration/Configuration.md (hunk 1 above)
Comment (MariekeGO) [reply user-guide/Standard_Apps/IDP/Advanced_configuration/Configuration.md:44]: OK to merge with my latest changes?

**Review by jaklke — COMMENTED** [review]
Perfect, thanks! Good to go.

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**